portfolio/src/test/java/com/google/sps/DataServletTest.java

```diff
@@ -38,19 +37,33 @@
 import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.junit.runners.JUnit4;
+import org.mockito.Mockito;
+import org.mockito.Mock;
+import org.mockito.MockitoAnnotations;
+import static org.mockito.Mockito.verify;
```

Comment:
these static import should be up with the others. mind running the formatter?

Reply:
done!